workers/unity/Packages/com.improbable.gdk.playerlifecycle/Systems/PlayerCreation/SendCreatePlayerRequestSystem.cs

```diff
@@ -12,21 +14,77 @@ public class SendCreatePlayerRequestSystem : ComponentSystem
     {
         private readonly EntityId playerCreatorEntityId = new EntityId(1);
 
-        private struct Data
+        private struct SendData
         {
             public readonly int Length;
             [ReadOnly] public ComponentDataArray<PlayerCreator.CommandSenders.CreatePlayer> RequestSenders;
             [ReadOnly] public ComponentDataArray<WorkerEntityTag> DenotesWorkerEntity;
             [ReadOnly] public ComponentDataArray<OnConnected> DenotesJustConnected;
         }
 
-        [Inject] private Data data;
+        [Inject] private SendData sendData;
+
+        private struct ResponseData
+        {
+            public readonly int Length;
+            [ReadOnly] public ComponentDataArray<PlayerCreator.CommandResponses.CreatePlayer> Responses;
+            [ReadOnly] public ComponentDataArray<WorkerEntityTag> DenotesWorkerEntity;
+        }
+
+        [Inject] private ResponseData responseData;
+
+        private readonly Dictionary<long, float> requestTimeMap = new Dictionary<long, float>();
```

Comment:
This is here only to test that request ids are passed properly.

Reviewers, can you think of a better way to test this?

Reply:
This isn't the place to test this imo

Reply:
But where is the better place to test this?

Reply:
In the test project that I'm going to build today 😉

Reply:
Let's raise a ticket

Reply:
Nice :D I'll clean this one up

Reply:
+1 this is quite a lot of code for doing something which isn't needed here. Especially as this is a module and not in playground.

Reply:
Fixed